api/apps/api/src/modules/projects/dto/geo-feature.mapper.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { GeoFeaturesService } from '../../geo-features/geo-features.service';
+import { PaginationMeta } from '../../../utils/app-base.service';
+import { GeoFeature } from '../../geo-features/geo-feature.api.entity';
+
+@Injectable()
+export class GeoFeatureMapper {
```

Comment:
what do you intend the role of these `Mapper` classes to be? right now they only provide serialization. do you suggest to possibly use these classes for other functionality going forward? I am also slightly confused by the naming as the (current) CRUD-ish service classes do implement a kind of mapper pattern. But besides naming choices, my question is mainly about the architectural intent of these `Mapper` classes.

Reply:
@hotzevzl `mapper` is designed to prepare response in the view layer (REST) and is solely aimed for clear responsibilities and separate from transport(access) layer - base service it strongly coupled for JSON Api format.

Reply:
ok, I agree with the separation. I am still not sure about the name, but I can live with it. would `*Serializer` not convey the intent better? or `*View`? `*ResourceView`? 🤔 
just minor of course, but while we're touching this, I'd try to settle on a solid naming convention - though it may be just me, and all of this is somewhat subjective in any case.

Reply:
@hotzevzl I am totally fine with any of those, no strong attachment to a particular one. `map` seems to be most adequate and common (mapping one to another) but I may be just "used to" it ;), while `serializing` is rather related to converting into byte stream.

so I will let you decide and adapt 💪

Reply:
it could also be `marshalling` ; )

few more words on `serializer` - indeed to relates to mapping an object into a format that can be stored/transmitted.

However `Nestjs` uses `serialization` as well https://docs.nestjs.com/techniques/serialization

So again - I am fine with any of those, let's make a choice and stick to it (may be some style/contribution guide?)

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/218/commits/0f09c1946f706fb33eea96931ef326ccd8d5c019